src/main/javac/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaComputeToVanillaConverter.java

```diff
@@ -50,6 +50,7 @@ public ToscaComputeToVanillaConverter(ManagementContext mgmt) {
         spec.configure(VanillaSoftwareProcess.LAUNCH_COMMAND, "true");
         spec.configure(VanillaSoftwareProcess.STOP_COMMAND, "true");
         spec.configure(VanillaSoftwareProcess.CHECK_RUNNING_COMMAND, "true");
+        spec.configure("tosca.template.id", id);
```

Comment:
Cleaner to make tosca.template.id a config key and use it here and in ToscaPlanToSpecTransformer.